src/PasswordResetController.js

```diff
@@ -125,7 +125,7 @@ export default FormController.extend({
     });
 
     if (!this.settings.get('features.hideBackToSignInForReset')) {
-      this.addFooter(FooterSignout);
+      this.addFooter(FooterSignout, { linkText: loc('signout', 'login'), linkClassName: 'goto', closeSession: true });
```

Comment:
why passwordReset has special ?

Reply:
`FooterSignout` is used in 3 controllers: RecoveryChallenge, RecoveryQuestion, PasswordReset.
For first 2 ones `FooterSignout` renders 'Back' link and does not clear session. For PasswordReset it renders 'Sign Out' and closes session.
If I'm not mistaken, in case of password reset user is already logged in, and in first 2 cases user is in recovery process and is not logged in

Reply:
A more general solution would be 
```
 if authClient.session.sessionExists() then
  authClient.session.closeSession();
```

The implication is it triggers 2 API calls.

@aarongranick-okta any thoughts?

Reply:
It would be good to make fewest XHR calls in this case. calling `sessionExists` does a GET on `/sessions/me` and `closeSession` does a DELETE on the same endpoint. I think we can probably just call `closeSession` and catch/ignore any error thrown from it.

Reply:
Fair enough.
It might trigger Splunk alert on the error when `closeSession` on page where there are no session at all.

Reply:
@denysoblohin-okta poking in because I'm working on a customer issue where on v1 password reset they expect only "Back to sign in" not "Sign out" even when they have the `signOutLink` customization because the user is not signed in so it doesn't make sense. This UX has also been confirmed by @vincentvoong-okta on[ this thread](https://okta.slack.com/archives/CCA77QVL2/p1619479941015500).

Reply:
@haishengwu-okta @sherizada-okta I've changed 'Sign out' to 'Back to sign in' for password reset flow.
The only flow that uses 'Sign out' link with closing session for v1 is password expiring.

Reply:
@haishengwu-okta "It might trigger Splunk alert on the error when closeSession on page where there are no session at all."

Do we know if it will or not? This seems fairly important. May be worth 2 xhrs to avoid this.

Reply:
@haishengwu-okta Added checking for session existence before session deletion